### PR TITLE
Standard names table update.

### LINKED ISSUE #4479: Update CF standard name table
Update this file: https://github.com/SciTools/iris/blob/main/etc/cf-standard-name-table.xml. New file can be downloaded from https://cfconventions.org/standard-names.html

### PR DESCRIPTION
**Pull request by pp-mo**
Closes #4479
Updated standard names, in advance of 3.2 release.

Was considering also the metadata translations, and https://github.com/SciTools/iris/issues/3950
But I have been reminded that standard-names is not really a part of that, so I'm going to deal with that [separately](https://github.com/SciTools/iris/pull/4484).

### CODE REVIEW

File: docs/src/whatsnew/latest.rst [hunk 1]
```diff
@@ -90,6 +90,9 @@ This document explains the changes made to Iris for this release
    benchmark scripts. Also added a regridding benchmark that uses this data
    (:pull:`4402`)
 
+#. `@pp-mo`_ updated the CF standard_names table to version 78.
```
Comment (bjlittle) [inline docs/src/whatsnew/latest.rst:93]:
```suggestion
#. `@pp-mo`_ updated to the latest CF Standard Names Table ``v78`` (21 Sept 2021).
```

File: docs/src/whatsnew/latest.rst [hunk 2]
```diff
@@ -90,6 +90,9 @@ This document explains the changes made to Iris for this release
...
+   (:issue: `4479`, :pull:`4483`)
```
Comment (bjlittle) [inline docs/src/whatsnew/latest.rst:94]:
```suggestion
   (:issue:`4479`, :pull:`4483`)
```

**Review by bjlittle — CHANGES_REQUESTED** [review]
@pp-mo Awesome thanks.

Verified this PR includes `v78` of the CF Standard Names Table as `XML`.

Just some suggested minor `whatsnew` tweaks, then we're there :+1:

**Comment by pp-mo** [comment]
Just committed your suggestions.
Hope that's all good !

**Review by bjlittle — APPROVED** [review]

**Pull request merged**